<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickParameterFilterDialogBase</class>
 <widget class="QDialog" name="PickParameterFilterDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Parameters to Study</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="TextLabel">
       <property name="text">
        <string>Select which parameters to display and focus on:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="toolTip">
        <string>Tell what to do with your selection</string>
       </property>
       <item>
        <property name="text">
         <string>Show All Parameters (filter disabled)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Failing Parameters Only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parameters with DataCleaning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parameters with Cp less than...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parameters with Cpk less than...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Specific names (eg: Iddq* )</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Selections in list...</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLimit">
       <property name="toolTip">
        <string>Enter Cp or Cpk limit...</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="TextLabelName">
       <property name="text">
        <string>Name string:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNameMask">
       <property name="toolTip">
        <string>Enter parameter name mask (use * and ? if needed). eg: Idd???.te*</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters matching filter:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidgetParameter">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Parameter#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
